content/guidedLearning/workshop/Bowling.md

```diff
@@ -0,0 +1,226 @@
+
+## Create the Scene
+
+Let’s first create a new playground. We can get rid of the sphere and ground within the scene since we later add our own assets. Before moving onto the next step, be sure that your code doesn’t include the sphere or ground.
+
+```javascript
+var createScene = function () {
+    // This creates a basic Babylon Scene object (non-mesh)
+    var scene = new BABYLON.Scene(engine);
+
+    // This creates and positions a free camera (non-mesh)
+    var camera = new BABYLON.FreeCamera("camera1", new BABYLON.Vector3(0, 5, -10), scene);
+
+    // This targets the camera to scene origin
+    camera.setTarget(BABYLON.Vector3.Zero());
+
+    // This attaches the camera to the canvas
+    camera.attachControl(canvas, true);
+
+    // This creates a light, aiming 0,1,0 - to the sky (non-mesh)
+    var light = new BABYLON.HemisphericLight("light", new BABYLON.Vector3(0, 1, 0), scene);
+
+    // Default intensity is 1. Let's dim the light a small amount
+    light.intensity = 0.7;
+
+    return scene;
+};
+```
+
+Since we’re creating the scene in the Playground, save the scene and copy the URL so that you can save incremental updates throughout the tutorial.
+
+## Enable Physics
+
+To bring our scene to life with realistic physics, Babylon.js provides a powerful function called `enablePhysics`. This essential function serves as the gateway to integrating physics into our scene. By invoking `enablePhysics`, we enable dynamic physics simulation for our objects, allowing them to respond to forces and collisions realistically. This function is necessary for creating a scene with physics because it sets the stage for objects to interact with each other in a lifelike manner. From simulating the movement of a bowling ball as it rolls down the lane to accurately calculating the collision response between the ball and the pins, `enablePhysics` is the key that unlocks the potential for immersive and interactive experiences.
+
+Let’s enable the physics engine just below the code for light intensity. The `enablePhysics` function takes two parameters: `gravity` and `plugin`. We create a variable for each parameter and pass them each into the function. For `gravity`, we use the vector `(0, -9.81,0)` which reflects real Earth gravity. As for the `plugin`, we assign the `HavokPlugin`.
+
+```javascript
+var gravityVector = new BABYLON.Vector3(0, -9.81, 0);
+var physicsPlugin = new BABYLON.HavokPlugin();
+scene.enablePhysics(gravityVector, physicsPlugin);
+```
+
+Now that physics is enabled, let’s create our bowling lane!
+
+## Create Bowling Lane
+
+The bowling lane is considered the ground in our scene. We want to provide a thin long platform for both the pins to sit and the bowling ball to roll. Using `MeshBuilder`, we create the lane and assign the width and height parameters, 6 and 20 respectively.
+
+```javascript
+var lane = BABYLON.MeshBuilder.CreateGround("lane", { width: 6, height: 20 }, scene);

```

Comment:
nit - const

Reply:
@RaananW if you could give me until EOD to make the update that'd be perfect. I'd also need to update the Playground that's linked as well. I'll commit my changes and @-mention you once it's ready for your review. Thanks for reviewing the tutorial!

Reply:
The PR was already approved. The minute you give me the OK i'll merge it. No stress :-)

Reply:
@RaananW could you please check again now to confirm if the changes meet your needs? we don't have such declarations in Python so thanks for your understanding haha!

Reply:
perfect thanks